docs/Users_Guide/overview.rst

```diff
@@ -223,7 +223,7 @@ METplus Components Python Requirements
       and much more easier
     -
   * - imageio
-    -
+    - >=2.19.3
```

Comment:
This is listed as 2.19.3 in plotpy requirements.txt, but 2.6.1 in calcpy requirements.txt.  Should we update the calcpy requirements.txt file?

Reply:
Created issue in METcalcpy to update the versions in the requirements.txt:
https://github.com/dtcenter/METcalcpy/issues/276